src/test/TxTests.cpp

```diff
@@ -603,6 +604,19 @@ manageData(std::string const& name, DataValue* value)
     return op;
 }
 
+Operation
+bumpSequence(AccountID const& account, SequenceNumber to, BumpSeqValidRange* range)
```

Comment:
this is different than other operation functions, as it contains AccountID const& account directly (no calling testAccount.op() on result here), but this operation only makes sense when AccountID is present

I think it is good that this is different.

Reply:
I figured that if the 'default behavior' is desired, one can always set account = tx.sourceAccount, but yes, this interface is slightly different.